letsencrypt/auth_handler.py

```diff
@@ -42,7 +42,7 @@ class AuthHandler(object):
         form of :class:`letsencrypt.achallenges.AnnotatedChallenge`
 
     """
-    def __init__(self, dv_auth, cont_auth, acme, account):
+    def __init__(self, dv_auth, cont_auth, acme, account, config=None):
```

Comment:
Also, I avoided doing that because many tests call ```AuthHandler``` without ```config```, which would be broken by requiring ```config```. Should I go on with that change and fix the tests?